spring-cloud-dataflow-server-core/src/main/java/org/springframework/cloud/dataflow/server/service/JobExplorerContainer.java

```diff
@@ -9,16 +9,18 @@
 import org.springframework.cloud.dataflow.schema.SchemaVersionTarget;
 import org.springframework.cloud.dataflow.schema.service.SchemaService;
 import org.springframework.cloud.dataflow.server.controller.NoSuchSchemaTargetException;
+import org.springframework.transaction.PlatformTransactionManager;
 import org.springframework.util.StringUtils;
 
 public class JobExplorerContainer {
 	private final Map<String, JobExplorer> container = new HashMap<>();
 
-	public JobExplorerContainer(DataSource dataSource, SchemaService schemaService) {
+	public JobExplorerContainer(DataSource dataSource, SchemaService schemaService, PlatformTransactionManager platformTransactionManager) {
```

Comment:
Who is injecting the `ResourcelessTransactionManager` ?

Reply:
Not sure that I understand.  We are not using ResourceslessTransactionManager.

Reply:
Sorry, who creates the tx mgr?

Reply:
It is the one used by SCDF 
https://github.com/spring-cloud/spring-cloud-dataflow/blob/main/spring-cloud-dataflow-server-core/src/main/java/org/springframework/cloud/dataflow/server/config/DataFlowServerConfiguration.java#L93C1-L100C3